controllers/gatekeeper_controller.go

```diff
@@ -1472,59 +1488,60 @@ func setContainerAttrWithFn(obj *unstructured.Unstructured, containerFn func(map
 	return nil
 }
 
-func setContainerArg(
-	obj *unstructured.Unstructured, argName string, argValue string, isMultiArg bool,
+// updateContainerArg takes an object, container flag argument name and value to
+// set on the object, and a boolean whether the flag should be deleted, and
+// returns any resulting errors.
+func updateContainerArg(
+	obj *unstructured.Unstructured, argName string, argValue string, remove bool,
 ) error {
 	return setContainerAttrWithFn(obj, func(container map[string]interface{}) error {
 		args, found, err := unstructured.NestedStringSlice(container, "args")
 		if !found || err != nil {
 			return errors.Wrapf(err, "Unable to retrieve container arguments for: %s", managerContainer)
 		}
 
-		exists := false
+		// Determine whether this is a flag that can be declared more than once.
+		multiArgFlags := []string{
+			"--operation",
+			"--disable-opa-builtin",
+			"--exempt-namespace",
+			"--exempt-namespace-prefix",
+			"--exempt-namespace-suffix",
```

Comment:
If gatekeeper updated, this list could be changed right?

Reply:
The multi-arg list could, yes. It's something we'd have to monitor.

Reply:
Oh Also metrics-backend can be multiple

Reply:
Oh shoot, you're right--I missed adding that one here.